requirements.txt

```diff
@@ -3,6 +3,7 @@ wooper
 PyRTF3
 xhtml2pdf
 werkzeug>=0.9.4,<=0.11.15
+urllib3<1.26
```

Comment:
It would be good to add this to `setup.py:install_requires` as well.

Reply:
done